pages/common/ibmcloud.md

```diff
@@ -0,0 +1,24 @@
+# ibmcloud
+
+> A command line tool for managing IBM Cloud apps and services.
+> More information: <https://ibm.biz/cli-docs>.
```

Comment:
<https://cloud.ibm.com/docs/cli?topic=cli-ibmcloud_cli> actually covers all of the command line options I think.

Reply:
Yes, you're right, I had some trouble finding reference for the general command in the docs. This is a way better link. The one I put in was from the CLI help page :-)